PR - fix : razorpay form in mobile view of header and addition of go to top button

sanjay7178 opened the pull request:
check preview : https://github.com/null-open-security-community/swachalit/issues/134#issue-2501037627

abhisek commented:
@sanjay7178 Can you rebase with `master` branch please? I see stuff that are already merged in #131

sanjay7178 commented:
@abhisek i rebased to upstream/master to this origin pr branch , but faced few issues 
as preview in #134 i wasn;t able to get css working right now as per my dev setup ie.. broke the css 
![image](https://github.com/user-attachments/assets/901a72b9-b2dd-4482-b68b-c5195ed85c73)

sanjay7178 commented:
> @abhisek i rebased to upstream/master to this origin pr branch , but faced few issues as preview in #134 i wasn;t able to get css working right now as per my dev setup ie.. broke the css ![image](https://private-user-images.githubusercontent.com/97831658/363754608-901a72b9-b2dd-4482-b68b-c5195ed85c73.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyOTY1OTgsIm5iZiI6MTcyNTI5NjI5OCwicGF0aCI6Ii85NzgzMTY1OC8zNjM3NTQ2MDgtOTAxYTcyYjktYjJkZC00NDgyLWI2OGItYzUxOTVlZDg1YzczLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAyVDE2NTgxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2NTI0ZWUyODI2NTM4MjU3OTBmZGNkNDA1MmRkOTM3M2M3OTExNzk0YzdjYWRkMjJhYmIzODg1ZjA0YzQ1YzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qaRLhilPjeI9qNwtfi_Ek4z2xwrG7778xc7xwGMq_FA)

fixed in https://github.com/null-open-security-community/swachalit/pull/135/commits/af089f691444c51849030a5b0d42dfd19b9e0131
